public/locales/fr/gamepage.json

```diff
@@ -33,7 +33,7 @@
         },
         "uninstall": {
             "checkbox": "Remove prefix: {{prefix}}{{newLine}}Note: This can't be undone and will also remove not backed up save files.",
-            "message": "Voulez-vous désinstaller ce jeu ?",
+            "message": "Voulez-vous désinstaller ce jeu\u202f?",
```

Comment:
remove these characters otherwise they will break the translations.

Reply:
Will do. I have no idea how this happened.

Reply:
Seems resolved after rebase, please confirm.